op-node/rollup/derive/engine_controller.go

```diff
@@ -360,12 +371,12 @@ func (e *EngineController) InsertUnsafePayload(ctx context.Context, envelope *et
 	}
 	// Insert the payload & then call FCU
 	status, err := e.engine.NewPayload(ctx, envelope.ExecutionPayload, envelope.ParentBeaconBlockRoot)
-	if err != nil {
+	if err != nil && !e.checkELSyncTriggered(status.Status, err) {
 		return NewTemporaryError(fmt.Errorf("failed to update insert payload: %w", err))
 	}
 
 	//process inconsistent state
-	if status.Status == eth.ExecutionInconsistent {
+	if status.Status == eth.ExecutionInconsistent || e.checkELSyncTriggered(status.Status, err) {
```

Comment:
In line 426, there is also a condition like `if status.Status == eth.ExecutionInconsistent`. Why is it not necessary to add `e.checkELSyncTriggered(status.Status, err)` there?
Additionally, the logic at line 426 is very similar to the logic here. I would like to know what their functions are.

Reply:
Initially, the intention here was to verify the synchronization status in two steps, hence the different logic checks. The updated commit added retry logic and moved the FCU status update into the first `if` condition, allowing the status verification to be completed in a single step. There are no longer two separate logic checks now.